### DISCUSSION TITLE
new mimeType in sunbird-sdk

### DISCUSSION THREAD

**Discussion by amantarento** [Sunbird-Ed/Community#670] [General Discussion]
I'd like to include a new 'mimeType' (application/json) to enable downloading content for offline use. How can we add this?

**Comment by swayangjit** [comment]
@amantarento right now  which API you want to use to download it ?

**Reply by amantarento** [reply]
@swayangjit  i am using this method this.contentService.importContent() .

**Reply by swayangjit** [reply]
@amantarento You can do it in the sunbird-mobile-sdk 
https://github.com/Sunbird-Ed/sunbird-mobile-sdk/blob/4e470d5f3637b442d4ec5317f89c6ab8e95b9a7c/src/content/impl/content-service-impl.ts#L395

https://github.com/Sunbird-Ed/sunbird-mobile-sdk/blob/4e470d5f3637b442d4ec5317f89c6ab8e95b9a7c/src/content/handlers/search-content-handler.ts#L301

You can add mimeType here.

**Comment by anisha147** [comment]
Hi @amantarento 

We hope your queries are answered. If yes, could you please close this thread from your end?

cc: @mohitga @swayangjit

**Reply by amantarento** [reply]
yes
